src/blueprints/tv2/value-objects/tv2-layers.ts

```diff
@@ -3,19 +3,30 @@ export enum Tv2SourceLayer {
   CAMERA = 'studio0_camera',
   JINGLE = 'studio0_jingle',
   AUDIO_BED = 'studio0_audio_bed',
+  OVERLAY = 'studio0_overlay',
+  IDENT = 'studio0_graphicsIdent',
+  LOWER_THIRD = 'studio0_graphicsLower',
   GRAPHICS_ACTION_COMMAND = 'studio0_adlib_graphic_cmd',
   AUDIO_ACTION_COMMAND = 'studio0_sisyfos_adlibs',
   DOWNSTREAM_KEYER_ACTION_COMMAND = 'studio0_dsk', // Original one from Blueprint includes the number of the DSK, followed by '_cmd'.
+  SELECTED_GRAPHICS_FULL = 'studio0_selected_graphicsFull',
+  WALL_GRAPHIC = 'studio0_wall_graphics',
+  TELEPHONE_GRAPHIC = 'studio0_graphicsTelefon',
+  PILOT_GRAPHICS = 'studio0_pilot',
+  PILOT_OVERLAY_GRAPHICS = 'studio0_pilotOverlay',
```

Comment:
Would be nice just move the ones we need.
`SELECTED_GRAPHICS_FULL`, `WALL_GRAPHIC`, `TELEPHONE_GRAPHIC` and `PILOT_OVERLAY_GRAPHICS` does not seem to be used.

Also, it's inconsistent whether the Enum name is called `GRAPHIC` or `GRAPHICS`.

Reply:
Have removed the unused ones that i had brought over.

There are currently two other unused enum values, which was last edited by you. 
Should i also remove those.

```typescript
export enum Tv2GraphicsLayer {
  ...
  GRAPHICS_LOCATORS = 'graphic_locators'
}
```

```typescript
export enum Tv2VideoClipLayer {
  VIDEO_CLIP_ENABLE_PENDING = 'server_enable_pending'
}
```

Reply:
You can leave those for now. I need the locators for SOF-1695 og and I am still fiddling with the necesity of the "enable_pending"